interface/syslink.h

```diff
@@ -71,6 +71,8 @@ void syslinkReset();
 #define SYSLINK_OW_READ 0x22
 #define SYSLINK_OW_WRITE 0x23
 
-#define SYSLINK_SYS_NRF_VERSION 0x30
+#define SYSLINK_SYS_NRF_VERSION      0x30
+#define SYSLINK_SYS_SHUTDOWN_REQUEST 0x31
+#define SYSLINK_SYS_SHUTDOWN_ACK     0x32
```

Comment:
I would add this command into the PM group since it is related to power-management and not a syslink 'meta' message (like the nRF version). This mean that it would become 0x15, 0x16 declared a bit lower down: https://github.com/bitcraze/crazyflie2-nrf-firmware/blob/f1a4a701b60300eb80187a5cb8ce3a79badb37f0/interface/syslink.h#L61-L67

Reply:
I can get behind that, I think my thinking was that it was not _directly_ battery related, but it is power mangement.